client/src/components/ScoreSheet.js

```diff
@@ -0,0 +1,56 @@
+import React, { Component } from 'react';
+import { connect } from 'react-redux';
+import { Table, Container } from 'semantic-ui-react';
+
+class ScoreSheet extends Component {
+	render() {
+		const { activities, users } = this.props
+		return (
+			<div style={{overflowX: 'scroll'}}>
+				<Table style={{width: '80%', margin: '2% auto'}} celled size='large' definition textAlign='center' unstackable>
+						<Table.Header>
+							<Table.Row>
+								<Table.HeaderCell />
+								{activities.map(activity => <Table.HeaderCell key={activity.name} verticalAlign='top'>
+																							{activity.name}<br />
+																						</Table.HeaderCell>)}
+								<Table.HeaderCell>Head to Head</Table.HeaderCell>
+								<Table.HeaderCell>Total</Table.HeaderCell>
+							</Table.Row>
+							<Table.Row>
+								<Table.HeaderCell />
+								{activities.map((activity, index) => <Table.HeaderCell key={activity.name + index}>{activity.value}</Table.HeaderCell>)}
+								<Table.HeaderCell />
+								<Table.HeaderCell />
+							</Table.Row>
+						</Table.Header>
+						<Table.Body>
```

Comment:
I dunno about the score breakdown. But I do think clicking on a user and seeing a modal pop-up with their score calculation would be really neat. 

You click on Sanda's name and a summary of his exercises and total score is displayed. I don't know how this should be broken down though. What do you guys think? It might be good to create as an issue and go back and forth on.

Reply:
Submitted issue 26